modules/nw-proxy-configure-object.adoc

```diff
@@ -39,33 +41,15 @@ proxies can be created.
 . Create a ConfigMap that contains any additional CA certificates required for
 proxying HTTPS connections.
 +
-NOTE: You can skip this step if the proxy’s identity certificate is signed by an
+[NOTE]
+====
+You can skip this step if the proxy’s identity certificate is signed by an
 authority from the RHCOS trust bundle.
-
-.. Create a file called `user-ca-bundle.yaml` with the following contents, and provide the values of your PEM-encoded certificates:
-+
-[source,yaml]
-----
-apiVersion: v1
-data:
-  ca-bundle.crt: | <1>
-    <MY_PEM_ENCODED_CERTS> <2>
-kind: ConfigMap
-metadata:
-  name: user-ca-bundle <3>
-  namespace: openshift-config <4>
-----
-<1> This data key must be named `ca-bundle.crt`.
-<2> One or more PEM-encoded X.509 certificates used to sign the proxy's
-identity certificate.
-<3> The ConfigMap name that will be referenced from the Proxy object.
-<4> The ConfigMap must be in the `openshift-config` namespace.
-
-.. Create the ConfigMap from this file:
+====
 +
 [source,terminal]
 ----
-$ oc create -f user-ca-bundle.yaml
+# oc create configmap user-ca-bundle --from-file=ca-bundle.crt=/path/to/my-ca.crt -n openshift-config
```

Comment:
Did the $ really need to change to # here? (I'm assuming not since # means that it has to be run by a root user)